cmd/authelia-scripts/cmd_docker.go

```diff
@@ -177,17 +182,21 @@ func deployManifest(docker *Docker, tag string, amd64tag string, arm32v7tag stri
 func publishDockerImage(arch string) {
 	docker := &Docker{}
 
-	travisBranch := os.Getenv("TRAVIS_BRANCH")
-	travisPullRequest := os.Getenv("TRAVIS_PULL_REQUEST")
-	travisTag := os.Getenv("TRAVIS_TAG")
-
 	if travisBranch == "master" && travisPullRequest == "false" {
 		login(docker)
 		deploy(docker, "master-"+arch)
 	} else if travisTag != "" {
+		patchTag := publishedTags.FindStringSubmatch(travisTag)[1]
```

Comment:
Test the array size before doing unchecked array accesses. The best way would be to call publishedTags.FindStringSubmatch(travisTag) only once and reuse the result both for checking the length and accessing the elements. If the size of the array is wrong => log.Fatal